readme.md

```diff
@@ -304,318 +410,227 @@ Whenever a test fails, RSpec will put an error message after that test which sho
-
-Unit testing always should come before functional testing. Functional testing is much less crucial. Unit testing is so important, that...
-
-## Employers give you major bonus points for it
-
-You'll see the term **test coverage** pop up pretty often. People are always aiming for "100% test coverage". If your app has 100% test coverage, that means every single method in your app has a unit test verifying that it works.
-
-For instance, while it's easy and free to write Salesforce apps, Salesforce will only add your app to its "app store" if you've obtained 100% test coverage, and Salesforce's developer team can run your tests and have them all pass.
-
-#### What are the reasons testing is so important? Why would employers love it so much?
+## Break (10 min)
 
-# Writing tests
+## Garnet Example (5 min)
 
-We've asked you to write user stories. Writing unit tests is a very similar process.
-
-When we think of "testing" we tend to think of something you do *after* you've created something. With unit tests, you're encouraged to write the tests *first* before you even start writing actual code.
-
-#### Turn and talk
-
-> Why would you write tests beforehand? Come up with as many reasons as you can. To get started, imagine if instead of a `readme.md` for Landlord you'd been given just the following in a `spec.rb` file:
+We use RSpec to test GArnet, the attendance/homework tracking app. Before any changes get pushed up to our live server, they have to pass all the tests -- an automated system rejects the changes if they don't pass.
 
-```
-describe Apartment do
-  it "has the class Apartment"
-  it "has a String for an address"
-  describe "#add_tenant" do
-    context "when the number of tenants" do
-      context "is less than the number of beds" do
-        it "adds a tenant"
-      end
-      context "is equal to the number of beds" do
-        it "does not add a tenant"
-      end
-    end
-  end
-end
-```
+[Here's what that looks like. Seem familiar?](https://travis-ci.org/ga-dc/garnet/builds/89503768#L241) Clearly there are a lot of tests that are just pending and don't do anything yet -- almost 60! These dramatically help us plan.
 
-### TDD
+## You-Do: Purrspec (20 min)
 
-You've probably already run into the situation of not knowing when an app is "done". You've also probably started an assignment or project only to find there are an overwhelming number of things to do. Then, on top of that, you have all that by-hand, manual testing to do as you make your app.
+**Instructions:**
 
-When you write tests first, you're creating a tidy little checklist for yourself of things to complete. The **goal of unit tests** is that **when all of the tests pass, your app is complete**.
+1. Fork and Clone the following repo:
+[Purrspec](https://github.com/ga-dc/purrspec)
```

Comment:
this is very similar to dog. can you think of another example that's a little closer to industry? maybe a shopping cart class with tests? I have bandwidth to help if you want